modules/tinymce/src/core/main/ts/textpatterns/core/BlockPattern.ts

```diff
@@ -38,9 +39,14 @@ const applyPattern = (editor: Editor, match: BlockPatternMatch): boolean => {
   const pattern = match.pattern;
   const rng = resolvePathRange(dom.getRoot(), match.range).getOrDie('Unable to resolve path range');
 
+  const isBlockFormatName = (name: string, formatter: Formatter): boolean => {
```

Comment:
Wasn't sure if we'd prefer this here, or outside the `applyPattern` function, or something else?